<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookFrame</class>
 <widget class="QFrame" name="AddressBookFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="m_addressBookView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_addAddressButton">
       <property name="text">
        <string>New address</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_deleteAddressButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete address</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_copyAddressButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copy address</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/copy</normaloff>:/icons/copy</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_copyPaymentIdButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copy PaymentID</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/copy</normaloff>
         <selectedoff>:/icons/copy</selectedoff>:/icons/copy</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_addAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>addClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_deleteAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>deleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>copyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyPaymentIdButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>copyPaymentIdClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressBookView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>addressDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addClicked()</slot>
  <slot>deleteClicked()</slot>
  <slot>copyClicked()</slot>
  <slot>copyPaymentIdClicked()</slot>
  <slot>addressDoubleClicked(QModelIndex)</slot>
 </slots>
</ui>
